{"name": "test_update_user_info[Update User Info - Firstname Above Max Value]", "status": "failed", "statusDetails": {"message": "AssertionError: FAILURE: \u001b[31massert 200 == 400\ntests\\test_api.py:181 in test_update_user_info() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1", "trace": "FAILURE: \u001b[31massert 200 == 400\ntests\\test_api.py:181 in test_update_user_info() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1"}, "attachments": [{"name": "log", "source": "6d406594-d718-42e2-96d0-c8c8047ce3fc-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'Case': 'Update User Info - Firstname Above Max Value', 'Test Type': 'Negative', 'Request_Body': {'firstName': 'uTmBRHBZysQtDNGTTXkCHhkcMCfuvjQHPjPEfdRsGLQUndLUBSH', 'lastName': 'cotoGVJqYubzHVARRbfxiUzgWt'}}"}], "start": 1670168776545, "stop": 1670168777265, "uuid": "60f8168e-f8b2-4350-b697-0182750c090d", "historyId": "abd4cdcbd59b4a7fffeb3fe541e68396", "testCaseId": "9b22326ca1aa84dd7b5876f09937fe1e", "fullName": "tests.test_api.TestApi#test_update_user_info", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "DESKTOP-EGG762M"}, {"name": "thread", "value": "11972-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}